### DISCUSSION TITLE
Access Cookies of Browser

### DISCUSSION THREAD

**Discussion by Momolem** [Voltstro-Studios/UnityWebBrowser#136] [Q&A]
Hi!
So far I am very impressed by this project.

Is there any way of accessing the localStorage of the browser? I found the option to enable/disable it but no way of accessing cookies.

**Comment by Voltstro** [comment]
UWB doesn't have anyway of getting local storage or cookies.

Getting cookies does look like something that CEF (and as such, UWB) could do, and could be implemented.
